File: backend-dotnet/Data/Daos/Tasks.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("tasks", Schema = "taskmanager")]
public class Tasks
{
	public Tasks(){}	

	[Key]
	[Column("pk")]
	public long Pk {  get; set; }
	[Column("task_id")]
	public string TaskId {  get; set; }
	[Column("title")]
	public string Title {  get; set; }
    [Column("description")]
	public string Description {  get; set; }
    [Column("state")]
	public string State {  get; set; }
    [Column("priority")]
	public string Priority { get; set; }
    [Column("created_on")]
	public DateOnly CreatedOn { get; set; }
    [Column("limit_date")]
	public DateOnly? LimitDate { get; set; }
    [Column("project_fk")]
	public long ProjectFk {  get; set; }
    [Column("user_fk")]
	public long? UserFk { get; set; }


}

File: backend-python/daos/tasks.py
from sqlalchemy import Column, BigInteger, String, Date 
from database import Base

class Tasks(Base):
    __tablename__ = "tasks"
    
    pk = Column(BigInteger, primary_key = True, autoincrement = True)
    task_id = Column(String(80), nullable = False)
    title = Column(String(120), nullable = False)
    description = Column(String, nullable = True)
    state = Column(String(20), nullable = False)
    priority = Column(String(20), nullable = False)
    created_on = Column(Date, nullable = False)
    limit_date = Column(Date, nullable = True)
    project_fk = Column(BigInteger, nullable = False)
    user_fk = Column(BigInteger, nullable = True)